elementary/clients/gcs/client.py

```diff
@@ -23,8 +23,10 @@ def create_client(cls, config: Config) -> Optional['GCSClient']:
     def send_report(self, local_html_file_path: str, remote_bucket_file_path: Optional[str] = None) -> bool:
         report_filename = bucket_path.basename(remote_bucket_file_path) if remote_bucket_file_path else path.basename(local_html_file_path)
         bucket_report_path = remote_bucket_file_path if remote_bucket_file_path else report_filename
+        logger.info("Pushing report file %s to bucket %s.", bucket_report_path, self.config.gcs_bucket_name)
         try:
             bucket = self.client.get_bucket(self.config.gcs_bucket_name)
+            logger.info("Successfully intialized client connection to gcs bucket: %s.", self.config.gcs_bucket_name)
```

Comment:
As this is more a debug log what do you think about making it a debug log using `logger.debug(...)`? using debug here means that it will be logged only to the `edr.log` log file but won't be logged to the screen.

Reply:
Done:
```
            logger.debug("Successfully intialized client connection to gcs bucket: %s.", self.config.gcs_bucket_name)

```